WebApiStart/Services/IBookService.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using WebApiStart.Model;
+
+namespace WebApiStart
+{
+    public interface IBookService
+    {
+        List<Book> GetBooks();
```

Comment:
try using async/await mechanism.
but to support multi-threading need to refactor entire solution